docs/Components/Checkbox.md

```diff
@@ -7,9 +7,9 @@ Checkboxes are most commonly used to give merchants a way to make a range of sel
 #### Vanilla
 
 ```js
-import {render, ExtensionPoint, Checkbox} from '@shopify/argo-admin';
+import {extend, ExtensionPoint, Checkbox} from '@shopify/argo-admin';
```

Comment:
I only update Checkbox doc for now. Other docs will be audited as part of this issue https://github.com/Shopify/app-extension-libs/issues/933 

cc @alanthai